react and coffeescript example

kulicuu opened the pull request:
This is intended simply as a quick and dirty example of getting React working nicely with SocketStream.  There is nothing especially interesting about the app itself at this point -- that may or may not change in the next little while, time depending.

Commit:
cleared node_modules

Commit:
experiments in expensive computationally ui stuff

Commit:
improved react component keying,

Commit:
array comprehension expression change

Commit:
testing html5's drag and drop api

Commit:
testing html5's drag and drop api

Commit:
testing various things

Commit:
updated readme

Commit:
updated readme

Commit:
...

kulicuu commented:
@arxpoetica 
After pulling and trying to merge from upstream I had a bunch of merge conflicts with the Ractive stuff-- my own fault.  I think I've set everything as you had it (actually manually copying raw from Github) -- let me know if any problems.  (thanks & sorry to have to check)

arxpoetica commented on ractive-koa/client/code/libs/jquery.min.js:
```diff
@@ -0,0 +1,2 @@
+/*! jQuery v@1.8.0 jquery.com | jquery.org/license */
+(function(a,b){function G(a){var b=F[a]={};return p.each(a.split(s),function(a,c){b[c]=!0}),b}function J(a,c,d){if(d===b&&a.nodeType===1){var e="data-"+c.replace(I,"-$1").toLowerCase();d=a.getAttribute(e);if(typeof d=="string"){try{d=d==="true"?!0:d==="false"?!1:d==="null"?null:+d+""===d?+d:H.test(d)?p.parseJSON(d):d}catch(f){}p.data(a,c,d)}else d=b}return d}function K(a){var b;for(b in a){if(b== ...
```
@kulicuu I removed jQuery from this folder.

arxpoetica commented:
Yeah, @kulicuu I'm confused why there are differences here. Most of these changes are old, so I think you might have messed things up. Fortunately, I'm working on a separate branch which has all the correct changes (at least locally).

I'll make sure to back up my work before merging back again.

Any idea why this is messed up?

kulicuu commented:
Yeah sorry I had a merge conflict when trying to pull your stuff (master) before redoing my pull request.  For some reason there were merge conflicts (I have no idea why as I never edited any of the Ractive files), and I apologise I just grabbed your files from the master branch to make sure -- wrote over.  You can merge again, and I'll pull again.  Hopefully should'nt be a problem anymore.

arxpoetica commented:
I think I know why. Possibly you didn't pull the changes from master first? I had merged some changes into master a day or two ago, so your version didn't have those changes. Would that apply?

kulicuu commented:
Yeah something like that.  I definitely screwed up the conflict resolution in my impatience.  Not because I changed anything (I didn't) , just because the delta/diff makes it look like I changed _everything_.  Again, super apologies.  I think next time I may do the pull with a `force` flag or something, because changes coming in from master wouldn't generally be changing the folder I'm working in.  
(p.s.  of course even if you hadn't had local copy everything would be retrievable by checking out a prior commit.  Just an inconvenience.)

arxpoetica commented:
No worries. Just sorting it out. ;)
